etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/RuleDefinition.java

```diff
@@ -0,0 +1,9 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine;
+
+import java.util.List;
+
+public class RuleDefinition {
```

Comment:
Looks like you could merge `RuleDefinition` with `ValidationRule`.  Basically this could `implements Rule`?

Reply:
Looks like this one was fixed